tools/docs/config.ts

```diff
@@ -175,6 +220,11 @@ export async function generateConfig(dist: string, bot = false): Promise<void> {
       configOptionsRaw[headerIndex] +=
         `\n${option.description}\n\n` +
         genTable(Object.entries(el), option.type, option.default);
+
+      if (el.experimental && headerIndex !== -1) {
+        const footerIndex = findFooterIndex(configOptionsRaw, headerIndex);
```

Comment:
should be restored to not to depend on such strange computation. 🤔

Reply:
without it the warning message will be added between the table and the markdown description :/.
is there another way to find the to be footer location?

Reply:
A better and maybe more elegant solution would be to have a `<!-- prettier-ignore -->` in the markdown file as a search token for the warning message. 
the only downside is that we will have to make sure it exists in future experimental options.